src/extension.ts

```diff
@@ -338,24 +340,28 @@ function registerOpenShiftLocalCluster(
   extensionContext: extensionApi.ExtensionContext,
   telemetryLogger: extensionApi.TelemetryLogger,
 ): void {
-  const status = () => crcStatus.getConnectionStatus();
   const apiURL = 'https://api.crc.testing:6443';
   const kubernetesProviderConnection: extensionApi.KubernetesProviderConnection = {
     name,
     endpoint: {
       apiURL,
     },
-    status,
+    status: () => 'stopped',
```

Comment:
```suggestion
    status: () => crcStatus.getConnectionStatus()
```

otherwise for for a usecase when Openshift is Running it is going to be in a Runnign state on dashboard and Off in Resources settings.

Reply:
you're right. I need to fix the `getConnectionStatus` then bc after the initialize its state is unknown when it should be stopped

Reply:
fixed. I updated the status assigned during the setup. Not sure why it was `starting`, I updated it to `stopped`. The VM is actually `starting` when the start command is executed, not during the setup. Now it should work as expected.